dvc.yaml

```diff
@@ -1,14 +1,31 @@
 plots:
-  - acc_v_loss:
+  - Accuracy:
       x: step
       y:
-        training/plots/metrics/acc.tsv: acc
-        training/plots/metrics/loss.tsv: loss
+        training/plots/metrics/acc_train.tsv: acc_train
+        training/plots/metrics/acc_test.tsv: acc_test
+      y_label: accuracy
+  - Loss:
+      x: step
+      y:
+        training/plots/metrics/loss_train.tsv: loss_train
+        training/plots/metrics/loss_test.tsv: loss_test
+      y_label: loss
   - training/plots/images
   - training/plots/sklearn/confusion_matrix.json:
       template: confusion
       x: actual
       y: predicted
+  - training/plots/sklearn/confusion_matrix.json:
+      template: jitter.json
+      x: actual
+      y: predicted
+      title: Actual vs Predicted
```

Comment:
@dberenbaum it looks like you can only use the dataset for one plot/template:

<img width="1184" alt="image" src="https://user-images.githubusercontent.com/37993418/202954713-1d6a7063-79ea-4cb4-badc-2ad064c94d9b.png">

Version of DVC used: `2.34.2`

Reply:
The jitter template also seems to tank the extension's performance. Is there any way that we can change the template and aggregate the data before it is output by `plots diff`? (it also causes so less noticeable issues in the HTML shown above)

Reply:
> The jitter template also seems to tank the extension's performance. 

Do we know the reason for this? 

Confusion matrix is still needed, yep.

Reply:
> Do we know the reason for this?

I assume the reason is that we are not pre-processing the raw data. That means we are passing 10k+ data points for each revision. We are also rendering all of those points in a single plot.

Chrome even struggles with this plot:

https://user-images.githubusercontent.com/37993418/202976513-e851451c-9123-4dc7-ba29-1e61846a01b9.mov

Reply:
> it looks like you can only use the dataset for one plot/template

Fixed.

> we are plotting 10k+ data points

Downsampled to 500 points, although no guarantee users will do the same.

Reply:
@dberenbaum thanks! should bring the confusion matrix back?

Reply:
Yup, everything should be there. The problem was having the same ID for different plots. Making a note to provide a better warning/error for that.

<img width="1227" alt="Screenshot 2022-11-21 at 3 30 33 PM" src="https://user-images.githubusercontent.com/2308172/203152598-f0e6ea61-1b1d-4063-a14c-b08c44608e82.png">